packages/components/copybutton/src/CopyButton.tsx

```diff
@@ -13,7 +13,7 @@ import { cx } from 'emotion';
 
 export type CopyButtonProps = Omit<
   ButtonProps,
-  'children' | 'endIcon' | 'startIcon' | 'isDisabed' | 'size'
+  'children' | 'endIcon' | 'onCopy' | 'startIcon' | 'isDisabed' | 'size'
```

Comment:
Seems like a valid event, 🤔 why removing it?

Reply:
The CopyButton is overwriting it with it's own `onCopy` that we're using.